src/test/java/org/opensearch/security/ssl/util/SSLConfigConstantsTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+package org.opensearch.security.ssl.util;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import org.opensearch.common.settings.Settings;
+
+import static org.opensearch.security.ssl.util.SSLConfigConstants.SECURITY_SSL_HTTP_ENABLED_PROTOCOLS;
+import static org.opensearch.security.ssl.util.SSLConfigConstants.SECURITY_SSL_TRANSPORT_ENABLED_PROTOCOLS;
+import static org.junit.Assert.assertArrayEquals;
+
+public class SSLConfigConstantsTest {
```

Comment:
Thanks for adding this test, what do you think about checking that a client cannot connect via TLSv1 by default - such as the SSLTest.java tests

Reply:
The client can't establish connection which uses TLS1 or TLS 1.1 since for JDK in version 11.11 and 1.8 TLSv1 and TLSv1 were deprecated  as result you need to change `$JAVA_HOME/conf/security/java.security` to use it.
e.g. https://www.petefreitag.com/blog/tlsv1-tlsv1-1-disabled-java/